pages/clustering/high-availability.mdx

```diff
@@ -361,6 +361,9 @@ for which the timeout is used is the following:
 - FrequentHeartbeatReq -> coordinator sending to data instances
 - HeartbeatReq -> main sending to replica
 - TimestampReq -> main sending to replica
+- SystemHeartbeatReq -> main sending to replica
+- ForceResetStorageReq -> main sending to replica. The timeout is set to 60s.
+- SystemRecoveryReq -> main sending to replica. The timeout set to 5'.
```

Comment:
> Timeout configured to 10'' for system heartbeat, 60'' for force reset and 5' for system recovery.

Should you add for SystemHeartbeatReq? Also, can you stick to 's' (10s) instead of ' or mixing?

Reply:
I wrote above that by default timeout is 10s